src/slack_client.py

```diff
@@ -157,6 +158,8 @@ def message_send():
                 integrations.append(GroupIntegration)
             if "prlinks" in active_services:
                 integrations.append(PRLinkIntegration)
+            if "issues" in active_services:
+                integreations.append(IssueIntegration)
```

Comment:
![image](https://user-images.githubusercontent.com/18374604/86829282-1541c980-c049-11ea-9eab-b2e5ee710021.png)

Reply:
you know what i tried to grow past autocorrect but clearly i still require training wheels